overviewer_core/world.py

```diff
@@ -606,61 +621,28 @@ def __init__(self, regiondir, rel):
             'minecraft:red_shulker_box': (218, 14),
             'minecraft:black_shulker_box': (218, 15),
-            'minecraft:white_glazed_terracotta': (220, 0),
-            'minecraft:orange_glazed_terracotta': (221, 0),
-            'minecraft:magenta_glazed_terracotta': (222, 0),
-            'minecraft:light_blue_glazed_terracotta': (223, 0),
-            'minecraft:yellow_glazed_terracotta': (224, 0),
-            'minecraft:lime_glazed_terracotta': (225, 0),
-            'minecraft:pink_glazed_terracotta': (226, 0),
-            'minecraft:gray_glazed_terracotta': (227, 0),
-            'minecraft:silver_glazed_terracotta': (228, 0),
-            'minecraft:cyan_glazed_terracotta': (229, 0),
-            'minecraft:purple_glazed_terracotta': (219, 0),
-            'minecraft:blue_glazed_terracotta': (231, 0),
-            'minecraft:brown_glazed_terracotta': (232, 0),
-            'minecraft:green_glazed_terracotta': (233, 0),
-            'minecraft:red_glazed_terracotta': (234, 0),
-            'minecraft:black_glazed_terracotta': (235, 0),
-
-            'minecraft:white_stained_glass': (241, 0),
-            'minecraft:orange_stained_glass': (241, 1),
-            'minecraft:magenta_stained_glass': (241, 2),
-            'minecraft:light_blue_stained_glass': (241, 3),
-            'minecraft:yellow_stained_glass': (241, 4),
-            'minecraft:lime_stained_glass': (241, 5),
-            'minecraft:pink_stained_glass': (241, 6),
-            'minecraft:gray_stained_glass': (241, 7),
-            'minecraft:light_gray_stained_glass': (241, 8),
-            'minecraft:cyan_stained_glass': (241, 9),
-            'minecraft:purple_stained_glass': (241, 10),
-            'minecraft:blue_stained_glass': (241, 11),
-            'minecraft:brown_stained_glass': (241, 12),
-            'minecraft:green_stained_glass': (241, 13),
-            'minecraft:red_stained_glass': (241, 14),
-            'minecraft:black_stained_glass': (241, 15),
+            'minecraft:white_glazed_terracotta': (235, 0),
+            'minecraft:orange_glazed_terracotta': (236, 0),
+            'minecraft:magenta_glazed_terracotta': (237, 0),
+            'minecraft:light_blue_glazed_terracotta': (238, 0),
+            'minecraft:yellow_glazed_terracotta': (239, 0),
+            'minecraft:lime_glazed_terracotta': (240, 0),
+            'minecraft:pink_glazed_terracotta': (241, 0),
+            'minecraft:gray_glazed_terracotta': (242, 0),
+            'minecraft:silver_glazed_terracotta': (243, 0),
+            'minecraft:cyan_glazed_terracotta': (244, 0),
+            'minecraft:purple_glazed_terracotta': (245, 0),
+            'minecraft:blue_glazed_terracotta': (246, 0),
+            'minecraft:brown_glazed_terracotta': (247, 0),
+            'minecraft:green_glazed_terracotta': (248, 0),
+            'minecraft:red_glazed_terracotta': (249, 0),
+            'minecraft:black_glazed_terracotta': (250, 0),
```

Comment:
Can we handle these in the "for each color" block?
`self._blockmap['minecraft:%s_glazed_terracotta' % colors[i]] = (235 + i, 0)`

Reply:
Yes, we could handle this one in a loop.  I took it out just to be a little clearer and not be tempted into the the trap mentioned here: https://github.com/gmcnew/Minecraft-Overviewer/pull/4/files/bd78d7cd8352a45aaa38fecbfb121c20597ed6e4#r212585889